docs/src/man/reshaping_new.md

```diff
@@ -0,0 +1,498 @@
+# Reshaping Data
+
+*DataFrames.jl* is a popular julia library for tabular data which makes easier to read and transform data.
+It provides the abstractions of data frames and series, similar to those in *R*, and *Pandas*.
+
+In *DataFrames.jl* data reshaping means the transformaton of the structure of the table or vector (for example 
+data frames or series) to make it suitable for further analysis. Let us assume we have a data frame with 
+multiindices on the rows and columns. In `DataFrames.jl` currently we have two functions: `stack`, and `unstack`
+that we can use for reshaping our data. Their goals are very simple:
+- `stack` allows us to go from wide to long data format;
+- `unstack` allows us to go from long to wide data format. 
+
+!!! note
+
+  Parts of this section has been taken from [Bogumił Kamiński](https://bkamins.github.io/julialang/2021/05/28/pivot.html)
+
+First we will have a look that how to create multiple indices. There is a simplest way to create a multi index 
+data frame object is by passing a list of two or more arrays to the constructor. Have a look to the below example:
+
+```jldoctest dataframe
+julia> using DataFrames
+
+julia> grocessary = DataFrame(year = repeat(2019:2021, inner=4),
+                                list = repeat(["Sugar", "Soap", "Fruits", "Oil"], outer=3),
+                                month=1:12, Date=15:26)
+12×4 DataFrame
+ Row │ year   list    month  Date
+     │ Int64  String  Int64  Int64
+─────┼─────────────────────────────
+   1 │  2019  Sugar       1     15
+   2 │  2019  Soap        2     16
+   3 │  2019  Fruits      3     17
+   4 │  2019  Oil         4     18
+   5 │  2020  Sugar       5     19
+   6 │  2020  Soap        6     20
+   7 │  2020  Fruits      7     21
+   8 │  2020  Oil         8     22
+   9 │  2021  Sugar       9     23
+  10 │  2021  Soap       10     24
+  11 │  2021  Fruits     11     25
+  12 │  2021  Oil        12     26
+
+julia> costs = select(grocessary, :year, :list, [:month, :Date] .=> x -> x/2,
+                             renamecols=false)
+12×4 DataFrame
+ Row │ year   list    month    Date
+     │ Int64  String  Float64  Float64
+─────┼─────────────────────────────────
+   1 │  2019  Sugar       0.5      7.5
+   2 │  2019  Soap        1.0      8.0
+   3 │  2019  Fruits      1.5      8.5
+   4 │  2019  Oil         2.0      9.0
+   5 │  2020  Sugar       2.5      9.5
+   6 │  2020  Soap        3.0     10.0
+   7 │  2020  Fruits      3.5     10.5
+   8 │  2020  Oil         4.0     11.0
+   9 │  2021  Sugar       4.5     11.5
+  10 │  2021  Soap        5.0     12.0
+  11 │  2021  Fruits      5.5     12.5
+  12 │  2021  Oil         6.0     13.0
+```
+
+Now before moving further let's practice a bit basic data transformation skills:
```

Comment:
this comment is not clear. I thought the point of this section is to discuss `stack`.